src/main/scala/mesosphere/marathon/state/RootGroup.scala

```diff
@@ -408,12 +410,27 @@ object RootGroup {
     RootGroup(group.apps, group.pods, group.groupsById, group.dependencies, group.version)
   }
 
-  def rootGroupValidator(enabledFeatures: Set[String]): Validator[RootGroup] = {
-    noCyclicDependencies and
-      Group.validGroup(PathId.empty, enabledFeatures) and
-      ExternalVolumes.validRootGroup()
+  def rootGroupValidator(config: MarathonConf): Validator[RootGroup] = validator[RootGroup] { rootGroup =>
```

Comment:
What do you think about renaming this to `validRootGroup`?

Reply:
Done